gpflow/expectations_quadrature.py

```diff
@@ -62,7 +62,7 @@ def get_eval_func(obj, feature, slice=np.s_[...]):
 
 @dispatch(Gaussian, object, (InducingFeature, type(None)), object, (InducingFeature, type(None)))
 def _expectation(p, obj1, feature1, obj2, feature2, H=40):
```

Comment:
I don't see anywhere explicitly passing the desired number of `gauss_hermite_points`. If I'm not mistaken this function is always called with the default 40 points.

@vdutor can you verify whether I'm correct or not?

Reply:
Correct. In the unit tests and elsewhere we always use the default value of 40.